docs/src/pages/[platform]/connected-components/authenticator/headless/index.page.mdx

```diff
@@ -18,6 +18,6 @@ export async function getStaticProps() {
   return { props: {} }
 }
 
-<Fragment useCommonWebContent platforms={['web', 'flutter']}>
+<Fragment useCommonWebContent platforms={['web', 'react-native']}>
```

Comment:
Instead of using this fragment to conditionally render one of two files that are exactly the same, maybe we could just include all the code from `headless-usage.web.mdx` in this `index.page.mdx` file, and then delete `headless-usage.web.mdx`. Thoughts?

Reply:
Done